frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,42 @@
-import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
+import React, {useState, useEffect} from 'react'
+import { Link } from "react-router-dom"
+import './SCSS/Comment.scss'
+export const Comment = (props) => {
+    const [timePosted, setTimePosted] = useState(0);
+    const [timeFormat, setTimeFormat] = useState("sec");
+    
+    useEffect(() => {
+        timeElapseSincePosted();
+    }, [])
```

Comment:
i think you can omit the `[]`

Reply:
I think useEffect require at the very least an empty array as 2nd or else it would undefined which would cause issue I believe.

Reply:
ah ok seems like passing an empty array makes `useEffect` act as `componentDidMount` in contrast to `componentDidUpdate` (in the case second argument is non-empty array or undefined)